### PR TITLE
#104 append toi mdib report retrofit

### PR DESCRIPTION
**Pull request by riechkathrin**

### CODE REVIEW

**Comment by JavierEspina** [comment]
_R1007: "Messages (expect for Discovery messages) send between a <<actor_somds_consumer>> and a <<actor_somds_provider>> SHALL be sent via HTTP."
Didn't I make a mistake in the Luebeck minutes (that has propagated through) by writing HTTP? I think we actually meant TCP.
Moreover the first verb should be in past tense.
Proposed rephrase: "_Messages (expect for Discovery messages) sent between a <<actor_somds_consumer>> and a <<actor_somds_provider>> SHALL be sent via a TCP connection."

**Comment by JavierEspina** [comment]
I think the note under R1008 should be moved out of that requirement box, as it applies to all 8 requirements in the section.
Also the note text (admittedly mine... created while typing minutes in real time) could be made a bit more precise / formal...

**Review by JavierEspina — CHANGES_REQUESTED** [review]
see last commit (by me) with the changes proposed above and feel free to roll them back, accept them or add on top of them

**Review by JavierEspina — COMMENTED** [review]
The changes requested earlier have been incorporated with the last commits. With that, it looks good to me. I propose that David does the final check and, if he sees no further issues, feels free to accept the PR.

File: SDPi_Supplement/asciidoc/volume3/mdib-report-retrofit/tf3-ch-8.2.8-mdib-report-retrofit.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+.R1010
+[sdpi_requirement#r1010,sdpi_req_level=shall]
+****
+A <<actor_somds_provider>> shall not add pm:AbstractDescriptor elements to pm:AbstractDescriptors below msg:DescriptionModificationReport/msg:ReportPart.
+
+.Notes
+[%collapsible]
+====
+NOTE: Replaces biceps:R5024: "Descriptors in this list shall not include nested descriptors."
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume3/mdib-report-retrofit/tf3-ch-8.2.8-mdib-report-retrofit.adoc:72]: As this cites the BICEPS standard, shall is deliberately written in capitals
```suggestion
NOTE: Replaces biceps:R5024: "Descriptors in this list SHALL not include nested descriptors."

```

**Review by d-gregorczyk — APPROVED** [review]

**Review by d-gregorczyk — APPROVED** [review]

**Pull request merged**